scripts/compat/run_python.py

```
@@ -0,0 +1,55 @@
+    _, hard = resource.getrlimit(resource.RLIMIT_CORE)
+    resource.setrlimit(resource.RLIMIT_CORE, (hard, hard))
+    child = subprocess.Popen([interpreter, script])
+    code = child.wait()
+    manifest.update(pid=child.pid, returncode=code, host=platform.platform())
+    (artifacts / f"probe-{version}-{child.pid}.json").write_text(json.dumps(manifest, indent=2) + "\n")
+    if code < 0:
+        print(f"native probe died from signal {-code}; pid={child.pid}", file=sys.stderr)
+        core = artifacts / f"core.{child.pid}"
+        if core.exists():
+            with (artifacts / f"backtrace-{child.pid}.txt").open("w") as output:
+                subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)